.vscode/launch.json

```diff
@@ -6,8 +6,42 @@
     "configurations": [
 
 
-
-
+    {
+        "name": "(gdb) pfu",
+        "type": "cppdbg",
+        "request": "launch",
+        "program": "${workspaceFolder}/cmake-build-debug/bin/pfu",
+        "args": [],
+        "stopAtEntry": false,
+        "cwd": "${workspaceFolder}",
+        "environment": [
+            { 
+                "name" : "MUD_DATA_DIR",
+                "value" : "${env:HOME}/tmp/pfiles"
```

Comment:
Typically when manually running it locally, I'd point it at a copy of the prod pfiles that live here.